src/libraries/System.Globalization/tests/CultureInfo/CultureInfoParent.cs

```diff
@@ -16,10 +16,20 @@ public class CultureInfoParent
         [InlineData("zh-HK", "zh-Hant")]
         [InlineData("zh-MO", "zh-Hant")]
         [InlineData("zh-TW", "zh-Hant")]
+        [InlineData("zh-Hans-CN", "zh-Hans")]
+        [InlineData("zh-Hant-TW", "zh-Hant")]
         public void Parent(string name, string expectedParentName)
         {
-            CultureInfo culture = new CultureInfo(name);
-            Assert.Equal(new CultureInfo(expectedParentName), culture.Parent);
+            try
+            {
+                CultureInfo culture = new CultureInfo(name);
+                Assert.Equal(new CultureInfo(expectedParentName), culture.Parent);
+            }
+            catch (CultureNotFoundException)
+            {
+                // on downlevel Windows versions, some cultures are not supported e.g. zh-Hans-CN
```

Comment:
Nit: Something to consider here, maybe assert we are running on a downlevel version of Windows?

Reply:
I am not worried about that as if this happen with any of these cultures will show up in many other places in our tests.

Reply:
Sounds good.